parsers/NL.py

```diff
@@ -211,5 +224,68 @@ def get_production_data_energieopwek(date, session=None):
 
     return df
 
+def get_wind_capacity_at(date: datetime, wind_capacity_df: pd.DataFrame) -> float:
+    in_service_mask = ((wind_capacity_df["Start date"] <= date) & (date <= wind_capacity_df["End date"])) | ((wind_capacity_df["Start date"] <= date) & (wind_capacity_df["End date"].isnull()))
+    return np.array(wind_capacity_df[in_service_mask]["#"] * wind_capacity_df[in_service_mask].kW).sum() / 1000
+
+def get_wind_capacities() -> pd.DataFrame:
+    url_wind_capacities = "https://windstats.nl/windstats1/ajax/statsJSON.cfm?minKW=0&maxKW=999999&gemeente=&provincie="
+    r = requests.get(url_wind_capacities)
+    installations_df = pd.DataFrame(columns=["Project", "#", "Brand", "kW", "Municipality", "Province", "Start date", "End date"])
+    all_installations_data = r.json()["dataList"]
+    for row in all_installations_data:
+        entry = {
+            "Project": row[0].split(">")[1].split("</a")[0],
+            "#": row[1],
+            "Brand": row[2],
+            "kW": row[3],
+            "Municipality": row[4],
+            "Province": row[5],
+            "Start date": row[6],
+            "End date": row[8],
+        }
+        installations_df = installations_df.append(entry, ignore_index=True)
+    installations_df.kW = installations_df.kW.astype(float)
+    installations_df["#"] = installations_df["#"].astype(int)
+    installations_df["Start date"] = pd.to_datetime(installations_df["Start date"]).dt.tz_localize("utc")
+    installations_df["End date"] = pd.to_datetime(installations_df["End date"]).dt.tz_localize("utc")
+    return installations_df
+
+def get_solar_capacity_at(date: datetime, solar_capacity_df: pd.DataFrame) -> float:
+    latest_year = date.year
+    while True:
+        # Latest capacity for the year to date might not have been published yet, so revert back to latest known year
+        if solar_capacity_df[solar_capacity_df.index.year == latest_year]["capacity (MW)"].empty:
+            latest_year -= 1
+        else:
+            return float(solar_capacity_df[solar_capacity_df.index.year == latest_year]["capacity (MW)"][0])
+
+def get_solar_capacities() -> pd.DataFrame:
+    solar_capacity_base_url = "https://opendata.cbs.nl/ODataApi/odata/82610ENG/UntypedDataSet?$filter=((EnergySourcesTechniques+eq+%27E006590+%27))+and+("
```

Comment:
For this, we simply parse the json that populates the data of the preview table of CBS